concepts/system-model/index.md

```diff
@@ -305,19 +340,19 @@ To incentivise inclusion of a transaction in a block, EIP 1559 allows a transact
  
 #### Autonity EIP 1559 configuration
 
-Autonity maintains the Ethereum protocol settings for EIP 1559:
+Autonity sets an EIP 1559 configuration of:
 
-- _block gas target_ = `15M`, the pre EIP 1559 block gas limit
-- _block gas limit_ = `30M`, double the preceding block gas limit accommodating 50% elasticity
-- _base fee change denominator_ = `8`, limiting possible change in base fee to 12.5% each block.
-- _elasticity multiplier_ = `2`, setting targeted block usage at 50% of the block limit.
+- _block gas target_ = `20M`, the block gas usage targeted for a block
+- _block gas limit_ = the gas limit possible for a block is dynamic and can change (increase) by no more than 10% a block from the actual block gas usage of the parent block
```

Comment:
Unless I am missing something, I think that the block gas limit can increase or decrease based on the parent block

Reply:
Corrected to 'increase or decrease'.

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.